src/ui/carousel/carousel-synchronizer.ts

```diff
@@ -0,0 +1,123 @@
+import { setf } from '../../setf/setf';
+import { Carousel } from './carousel';
+
+/**
+ * Will synchronize the slide positions of multiple carousels.
+ *
+ * This can be useful if you have separate areas for assets, copy and/or
+ * navigation that all need to be synchronized together.
+ */
```

Comment:
Maybe you can add the example you have in examples/carousel.html to make this clearer.

Reply:
Done.